### ISSUE TITLE
Unable to export drawio diagram using the CLI

### ISSUE THREAD

**Issue by aacebedo** [NixOS/nixpkgs#250986]
### Describe the bug
It is impossible to export diagrams via the drawio CLI.
When running the export command it ends up with the following message:
`Error: input file/directory not found`

### Steps To Reproduce
1. create a drawio diagram
2. run `drawio  -x -f svg -o /tmp/foo.svg ./foo.drawio`

### Expected behavior
A svg file is output. 

### Screenshots
N/A

### Additional context
Installed from stable channel. Running wayland. The graphical version is working correctly

### Notify maintainers
@qyliss @darkonion0

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 system: `"x86_64-linux"`
 - host os: `Linux 6.4.8-xanmod1, NixOS, 23.05 (Stoat), 23.05.20230822.a16f7eb`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.5`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by alyssais** [comment]
Can't reproduce, I'm afraid.  I ran:

nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/tarball/a16f7eb -p drawio --run 'drawio -x -f svg -o /tmp/foo.svg ./stack.drawio'

And it worked fine.

**Comment by aacebedo** [comment]
Are you running wayland?

**Comment by alyssais** [comment]
Yes, sway 1.8.1.  Do you think that would make a difference to whether it finds the input file?

**Comment by DarkOnion0** [comment]
> Can't reproduce, I'm afraid. I ran: nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/tarball/a16f7eb -p drawio --run 'drawio -x -f svg -o /tmp/foo.svg ./stack.drawio' And it worked fine.

Me neither (I just took your command and changed the input file path). I'm also running sway 🤷.

**Comment by aacebedo** [comment]
Found the problem.
I suppose you are running using xwayland.
Try setting NIXOS_OZONE_WL to use the native wayland electron version. The parameters is not added correctly by wrapper so we have an issue.

**Comment by alyssais** [comment]
Yep, I can reproduce with NIXOS_OZONE_WL.

**Comment by DarkOnion0** [comment]
Indeed, I'm running XWayland and when I launched the app with `NIXOS_OZONE_WL=1` the app failed to convert to svg. This issue could be linked to the one you already open for drawio crashing under Wayland and more globally the whole mess around electron breaking. 
I will push the last update soon, let me know if it resolves the problem.

**Comment by Vaisakhkm2625** [comment]
for now as a work around, we can run it as a xwayland client right?
```
env -u WAYLAND_DISPLAY drawio ./hello.drawio.png
```

this is working for me... though it's blurry because it's using xwayland..

**Comment by fidgetingbits** [comment]
I just ran into this running a script I use to drive drawio headless and although I can live with `env -u WAYLAND_DISPLAY` for now, I wonder if the `drawio-headless` package should explicitly unset this variable for now? Since it's meant to be run headless there shouldn't be any negative effect if `WAYLAND_DISPLAY` is unset right? At least until someone can figure out what the actual underlying problem is and fix it properly?
